CHANGELOG.md

```diff
@@ -5,15 +5,16 @@ on how to contribute to Cucumber.
 
 ### New Features
 
- * 
+ *
 
 ### Changed
 
  *
 
 ### Fixed
 
- *
+ * Database cleaning silently fails when using database_cleaner v2 adapter gems
```

Comment:
should this say something like "no longer fails", or have I misunderstood this?

Reply:
Ah, I think I see. I was describing the bug, and you would prefer that I describe the bugfix?